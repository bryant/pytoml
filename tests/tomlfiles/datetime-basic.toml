dob = 1982-10-22T23:59:00Z
