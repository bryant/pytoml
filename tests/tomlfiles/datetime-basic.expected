{
        'dob': dt.strptime("1982-10-22T23:59:00Z", pytoml.ISO8601)
    }
